src/js/share.ts

```diff
@@ -0,0 +1,66 @@
+/* global document, navigator, window */
+import { CityId } from "./types";
+import { determineShareUrl } from "./cityId";
+/**
+ * Copy `value` to the user's clipboard
+ *
+ * @param string value
```

Comment:
You can get read of these types of docstring values. We were using them as a precursor to TypeScript. Now they're redundant 🎉

Ditto other places in this PR

Reply:
I got rid of them and the comments.  Not sure if you'd agree with that but I feel like both are redundant at this point.